src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -631,6 +628,19 @@ class IOpenGL_PhysicalDeviceBase : public IOpenGLPhysicalDeviceBase
 		m_features.largePoints = false; //TODO
 		m_features.alphaToOne = false; //TODO
 		
+		m_features.multiViewport = IsGLES ? m_glfeatures.isFeatureAvailable(COpenGLFeatureMap::NBL_OES_viewport_array) : true;
+
+		if (m_glfeatures.isFeatureAvailable(m_glfeatures.NBL_EXT_texture_filter_anisotropic))
+		{
+			GetIntegerv(GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT, &num);
+			m_glfeatures.MaxAnisotropy = static_cast<uint8_t>(num);
+			m_features.samplerAnisotropy = true;
+			m_properties.limits.maxSamplerAnisotropyLog2 = std::log2((float)m_glfeatures.MaxAnisotropy);
+		}
+		else m_glfeatures.MaxAnisotropy = 0u;
```

Comment:
anisotropy is now core in desktop GL 4.6

also the ladder goes:

    ARB
    EXT